docs/data/tool/config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Node dependencies
+ */
+const path = require( 'path' );
+
+const root = path.resolve( __dirname, '../../../' );
+
+module.exports = {
+	namespaces: {
+		core: {
+			title: 'WordPress Core Data',
+			// Figures out a way to generate docs for the dynamic actions/selectors
```

Comment:
The tense here implies that the following code "figures out a way" to do something, but I'm thinking this is actually a TODO, eg:

```js
// TODO: Figure out a way to generate docs for dynamic actions/selectors
```

Sounds like something best filed as a "code quality" issue 😆, but I'd at least tidy up this comment so it isn't confusing.